packages/terra-open-source-scripts/src/terra-cli/prepare-for-release/index.js

```diff
@@ -0,0 +1,11 @@
+const prepareForReleaseMonoRepoHandler = require('./prepareForReleaseMonoRepoHandler');
+const prepareForReleaseRepoHandler = require('./prepareForReleaseRepoHandler');
+const isMonoRepo = require('../../utils/isMonoRepo');
+
+const prepareForRelease = {
+  command: 'prepare-for-release',
+  describe: 'Prepare the project for release by updating the version and changelog of the project(s)',
+  handler: async () => (await isMonoRepo() ? prepareForReleaseMonoRepoHandler() : prepareForReleaseRepoHandler()),
```

Comment:
Same comment as below about using a flag to flip between mono-repo and non-mono-repo commands.

Reply:
@mjhenkes I thought I remember asking you about this and you leaned this way.  Do you have a strong preference?

Reply:
I'd rather keep it simple and detect if we're in a mono repo. I use this same logic in terra dev site to determine which files to pull from.

Reply:
Keeping it for now unless @emilyrohrbough or others feel strongly. If so, we can discuss